ansible/roles/ironic-enrolment/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Ensure Python requirements are installed
+  pip:
+    requirements: "{{ '/'.join([role_path, 'files', 'requirements.txt']) }}"
```

Comment:
did you try just 'requirements.txt' here?

Reply:
I did, yes. It seems like while Ansible magically checks the role's *files* directory for the source field in `copy`/`template` actions, it doesn't work in the `pip` module.